flatpak-pip-generator

```diff
@@ -0,0 +1,115 @@
+#!/usr/bin/env python3
```

Comment:
This file doesn't belong in the repository.

Reply:
Done!